qiskit_experiments/library/tomography/tomography_analysis.py

```diff
@@ -84,6 +86,8 @@ def _default_options(cls) -> Options:
 
         options.measurement_basis = None
         options.preparation_basis = None
+        options.measurement_qubits = None
```

Comment:
Not directly, since that information is in the individual circuit metadata for which clbits to marginalize too. This is only used for the fitters to get the correct matrices for those qubits.

Reply:
If this is not configurable value, i.e. automatically set by internal logic, perhaps we can move these stuff to protected instance variable? I often think of this when writing other classes.

Reply:
It could be moved to experiment metadata and not be an analysis option I suppose, though this makes analysis dependent on experiment metadata.